security/login-error.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Automattic\VIP\Security;
+
+function use_ambiguous_login_error( $error ) {
+	global $errors;
+	$err_codes = $errors->get_error_codes();
+
+	$err_types = [
+		'invalid_username',
+		'invalid_email',
+		'incorrect_password',
+	];
+
+	foreach ( $err_types as $err ) {
+		if ( in_array( $err, $err_codes, true ) ) {
+			$error = '<strong>ERROR</strong>: Invalid login name or password.';
```

Comment:
No biggie, but since we only care about any of those we can just break on the first one?
```suggestion
			$error = '<strong>ERROR</strong>: Invalid login name or password.';
			break;
```